packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue

```diff
@@ -1007,16 +1008,12 @@ async function getAutoMapping() {
 
 const initialize = async () => {
 	// Model configuration input
-	const {
-		modelConfiguration,
-		modelOptions,
-		modelPartUnits,
-		modelPartTypes,
-		modelParameters: modelParams
-	} = await setupModelInput(modelConfigId.value);
+	const { model, modelConfiguration, modelOptions, modelPartUnits, modelPartTypes } = await setupModelInput(
+		modelConfigId.value
+	);
 	modelConfig.value = modelConfiguration;
 	modelStateOptions.value = modelOptions;
-	modelParameters.value = modelParams ?? [];
+	modelParameters.value = !model ? [] : getParameters(model);
```

Comment:
Could we instead improve `getParameters` to return `[]` if `model` is null or undefined.

Reply:
I would argue no.

I think at some level, we need to have functions that act on concrete objects, if we constantly have to check if the parameters we pass in are null or undefined it becomes quite difficult to to write coherent code, or to debug problems once we have nested several levels of these type of logic.